src/inject/inject.js

```diff
@@ -9,10 +9,10 @@ chrome.extension.sendMessage({}, function() {
       clearInterval(readyStateCheckInterval);
       chrome.runtime.onMessage.addListener(routeCommand);
     }
-  }, 10);
+  }, 3);
 
   function routeCommand(request) {
-    card = $('.active-card');
+    card = $('.card-detail-window');
```

Comment:
Ah, good one! Let me see if that works because I couldn't find `.active-card` in the DOM when I tried.